### DISCUSSION TITLE
Way to output installed packages list

### DISCUSSION THREAD

**Discussion by yusancky** [typst/typst#5123] [Q&A]
Is there a way to output a list of installed packages (similar to `python -m pip freeze`)?

Or do I have any other good methods (like getting from folder names)?

**Comment by LordBaryhobal** [comment]
From https://github.com/typst/packages?tab=readme-ov-file#downloads
> The Typst compiler downloads packages from the `preview` namespace on-demand. Once used, they are cached in `{cache-dir}/typst/packages/preview` where `{cache-dir}` is
> - `$XDG_CACHE_HOME` or `~/.cache` on Linux
> - `~/Library/Caches` on macOS
> - `%LOCALAPPDATA%` on Windows

**Reply by yusancky** [reply]
Thanks, I had noticed that, so I mentioned reading from the file at that time. I'm more curious if there's a better way.
